dlp/snippets/inspect/inspect_test.go

```diff
@@ -681,3 +688,206 @@ func TestInspectTableWithCustomHotword(t *testing.T) {
 		t.Errorf("TestInspectTableWithCustomHotword got %q, want %q", got, want)
 	}
 }
+
+func TestInspectDataStoreSendToScc(t *testing.T) {
```

Comment:
From another sample?

Reply:
Again so sorry for that, This is because of unwanted commits/branch conflicts, Yes this is from another sample.